firebase.json

```diff
@@ -218,18 +218,18 @@
       { "source": "/brick-breaker", "destination": "https://codelabs.developers.google.com/codelabs/flutter-flame-brick-breaker", "type": 301 },
       { "source": "/catalog/samples", "destination": "https://flutter.github.io/samples/#", "type": 301 },
       { "source": "/catalog/samples/*", "destination": "https://flutter.github.io/samples/#", "type": 301 },
-      { "source": "/chat", "destination": "https://github.com/flutter/flutter/wiki/Chat", "type": 302 },
+      { "source": "/chat", "destination": "https://github.com/flutter/flutter/blob/master/docs/contributing/Chat.md", "type": 302 },
       { "source": "/custom-deferred-components", "destination": "https://github.com/flutter/flutter/wiki/Deferred-Components#custom-implementations", "type": 301 },
       { "source": "/deferred-components-wiki", "destination": "https://github.com/flutter/flutter/wiki/Deferred-Components", "type": 301 },
```

Comment:
These references to https://github.com/flutter/flutter/wiki/Deferred-Components are still here because that page will be merged with https://docs.flutter.dev/perf/deferred-components